src/Services/GTT/shared/GTT.Application/Requests/CreateSportRequestModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GTT.Application.Requests
+{
+    public class CreateSportRequestModel
```

Comment:
Missing field UserId

Reply:
Where can I get the field UserId from?

Reply:
userId is from jwt token, we didn't implement authorization yet, so just skip it for now